lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -794,20 +794,17 @@ bool ProxyRenderDelegate::getInstancedSelectionPath(
     MDagPath&                       dagPath) const
 {
 #if defined(WANT_UFE_BUILD)
-    if (_proxyShapeData->ProxyShape() == nullptr) {
-        return false;
+    // When point snapping, only the point position matters, so return the DAG path and avoid the
+    // UFE global selection list to be updated.
+    if (pointSnappingActive()) {
+        dagPath = _proxyShapeData->ProxyDagPath();
+        return true;
     }
 
-    if (!_proxyShapeData->ProxyShape()->isUfeSelectionEnabled()) {
+    if (!_proxyShapeData->ProxyShape() || !_proxyShapeData->ProxyShape()->isUfeSelectionEnabled()) {
         return false;
     }
 
-    // When point snapping, only the point position matters, so return false
-    // to use the DAG path from the default implementation and avoid the UFE
```

Comment:
The default implementation in VP2 assumes that the instance id of the hit item can be mapped to the instance index of the instanced DAG path. But in maya-usd we don't support instanced proxy shape at least for now, so we just override the default behavior and we don't really care about the mapping at all.